src/runner.nim

```diff
@@ -116,7 +116,8 @@ func simplifyPaths(s: var string) =
   ##   `/tmp/nim_test_runner/bob.nim(8, 18)`
   ## But we don't strip in:
   ##   `/nim/lib/pure/unittest.nim(647, 43)`
-  s = s.replace("/tmp/nim_test_runner/", "")
+  # s = s.replace("/tmp/nim_test_runner/", "")
```

Comment:
Why this change?

Reply:
This was just to show how it would look on the CI when something fails

Reply:
I will remove the commit

Reply:
Removed